Formula/ruby.rb

```diff
@@ -36,12 +30,12 @@ class Ruby < Formula
   # The exception is Rubygem security fixes, which mandate updating this
   # formula & the versioned equivalents and bumping the revisions.
   resource "rubygems" do
-    url "https://rubygems.org/rubygems/rubygems-2.6.14.tgz"
-    sha256 "406a45d258707f52241843e9c7902bbdcf00e7edc3e88cdb79c46659b47851ec"
+    url "https://rubygems.org/rubygems/rubygems-2.7.4.tgz"
```

Comment:
Ruby 2.5.0 bundles with RubyGems 2.7.3, if we still stay at RubyGems 2.6.14 it seems quite odd. Let's try again? cc @ilovezfs @DomT4

Reply:
Nothing has changed. This will cause Ruby installations that already have bundler installed to become unlinked.

Reply:
Not initially, no? Because this is a new API version?

```
=> /usr/local/bin/gem install bundler
=> brew pull https://github.com/Homebrew/homebrew-core/pull/22118
=> brew upgrade -sv ruby

==> Caveats
Emacs Lisp files have been installed to:
  /usr/local/share/emacs/site-lisp/ruby
==> Summary
🍺  /usr/local/Cellar/ruby/2.5.0: 3,323 files, 18.4MB, built in 3 minutes 3 seconds
```

Reply:
Right but what about all the other rubies you'll want to sync ...

Reply:
Also that's only because you're not installing bin/bundler …

Reply:
Yeah, I appended that I wasn't necessarily endorsing the idea. It remains somewhat annoying the linking step happens before rather than after `post_install` or we'd have alternative hacks open to us. I'll think on it a bit more, but feel free to hold `gem` where it is for now if you want to.

Reply:
Right but we know how to work around that as per `node`.

We could dump the contents of bin in libexec/bin and then forcibly overwrite anything in HOMEBREW_PREFIX/bin with those files in post_install. :tada:

Reply:
Aye. I wasn't sure how much more you wanted to adopt the `node` system, given that everyone in the world has more or less acknowledged what a giant dancing hack that is 😄. But I agree it is doable.

Reply:
I think the alternative is doing the entire gem installation `ruby setup.rb --prefix= ...` in post_install, which seems more awful.